glide.yaml

```diff
@@ -34,3 +34,5 @@ import:
   version: 91cfa479c814065e420cee7ed227db0f63a5854e
 - package: github.com/kiali/k-charted
   version: 378e9360b5d146e419ce8b6203b35b0fa1fcfb07
+- package: github.com/openshift/client-go
+  version: release-3.11
```

Comment:
Two comments:
- vender changes should be in separated commits / PRs to not mix them with logic (desirable).
- if this client also compatible with openshift 4 ?

Reply:
No idea, it's the same version as we currently use for Openshift object versions. I don't think making them unsynced makes sense.

Reply:
But I wonder if this has not problems with the 
```
 package: k8s.io/client-go
```
or the openshift one only works with the os specific objects ?

(Just asking I didn't test locally the PR yet).

Reply:
The OS client-go has only the types